client/my-sites/domains/map-domain/index.jsx

```diff
@@ -112,7 +112,8 @@ export class MapDomain extends Component {
 		// Instead, we add the mapping directly
 		if ( selectedSite.is_vip ) {
 			wpcom
-				.addVipDomainMapping( selectedSite.ID, domain )
+				.site( selectedSite.ID )
+				.addVipDomainMapping( domain )
```

Comment:
The `addVipDomainMapping` and `addDomainMapping` methods are recent additions (#55644, Aug 2021), and this kind of migration doesn't reduce the bundle size that much -- these methods remain centralized and bundled everywhere. Did you consider removing them from the `wpcom` package and doing inline `wpcom.req.post` calls?

Reply:
TIL these were recent additions. I've refactored them both to use `wpcom.req` directly in https://github.com/Automattic/wp-calypso/pull/57912/commits/c08a374fc2ea69b8b29c7d2f406da51917ada236.

Would appreciate another look when you get a chance.